source/develop/infra/infrastructure.html.md

```diff
@@ -5,17 +5,17 @@ authors: dcaroest, dneary, eedri, ekohl, eyale, herrold, knesenko, ngoldin, quai
   rmiddle
 ---
 
-# Infrastructure
+# Infrastructure 基础设施
```

Comment:
Which language is it translated to? I wonder if we should have a page per langugage.

Reply:
It is Chinese.

I don't see any page translated in other languages currently but that would be nice.